pkg/reconciler/taskrun/validate_resources.go

```diff
@@ -25,53 +25,13 @@ import (
 	"github.com/hashicorp/go-multierror"
```

Comment:
nit: should we rename this file since this no longer has anything to do with validating resources?

Reply:
renamed to `validate_taskrun`